### PR TITLE
Update new logic

### PR DESCRIPTION
**Pull request by nguyenduyhao1111**

### CODE REVIEW

File: week0001/fizz_buzz_haond.rb [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-end
-
+  x = 0
+  y = 0
+  while(x + 3 <= n || y + 5 <= n)
+    if(x+3<=y+5)
+        x+=3
+        puts "Fizz"
+    else
+        y+=5
```
Comment (vietlq) [inline week0001/fizz_buzz_haond.rb]: space around the operator `+=`

Comment (nguyenduyhao1111) [inline week0001/fizz_buzz_haond.rb]: thank :+1:

File: week0001/fizz_buzz_haond.rb [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
-#buzz(5) => 'buzz'
-#buzz(10) => 'buzz'
-def buzz(n)
-  p 'buzz' if n%5==0
-end
-
+  x = 0
+  y = 0
+  while(x + 3 <= n || y + 5 <= n)
+    if(x+3<=y+5)
```
Comment (vietlq) [inline week0001/fizz_buzz_haond.rb]: Space around operators: `+` and `<=`

Comment (nguyenduyhao1111) [inline week0001/fizz_buzz_haond.rb]: thank :+1:

**Pull request merged**